src/Exceptionless.Core/Jobs/WorkItemHandlers/UserMaintenanceWorkItemHandler.cs

```diff
@@ -57,15 +65,37 @@ public override async Task HandleItemAsync(WorkItemContext context)
         }
     }
 
-    private void NormalizeUser(User user)
+    private Task NormalizeUsersAsync(IReadOnlyCollection<User> users)
+    {
+        foreach (var user in users)
+        {
+            user.FullName = user.FullName.Trim();
+            string email = user.EmailAddress.Trim().ToLowerInvariant();
+            if (!String.Equals(user.EmailAddress, email))
+            {
+                Log.LogInformation("Normalizing user email address {EmailAddress} to {NewEmailAddress}", user.EmailAddress, email);
+                user.EmailAddress = email;
+            }
+        }
+
+        return _userRepository.SaveAsync(users);
+    }
+
+    private async Task ResendVerifyEmailAddressEmailsAsync(IReadOnlyCollection<User> users)
     {
-        user.FullName = user.FullName.Trim();
+        var unverifiedUsers = users.Where(u => !u.IsEmailAddressVerified).ToList();
+        if (unverifiedUsers.Count is 0)
+            return;
+
+        foreach (var user in unverifiedUsers)
+            user.MarkEmailAddressUnverified();
+
+        await _userRepository.SaveAsync(unverifiedUsers);
 
-        string email = user.EmailAddress.Trim().ToLowerInvariant();
-        if (!String.Equals(user.EmailAddress, email))
+        foreach (var user in unverifiedUsers)
         {
-            Log.LogInformation("Normalizing user email address {EmailAddress} to {NewEmailAddress}", user.EmailAddress, email);
-            user.EmailAddress = email;
+            Log.LogInformation("Resending verify email address email to {EmailAddress}", user.EmailAddress);
+            await _mailer.SendUserEmailVerifyAsync(user);
```

Comment:
It's a reminder to verify or you never get emails from us. Shouldn't be many

Reply:
No way we can do this. I am sure we get tons of people signing up and never verifying their email address. This will send an email to those people every single time. Not only does that cost us money, but we will get reported as spam as well.

Reply:
I felt it would help a user get engagement to either verify or to remove their account.

Reply:
We have the actions for a reason. I'm ok with us doing some maintenance task on our data, I'm not ok with it sending out a crapload of emails.

Reply:
If you want a task to resend a verification email to a single user then that is great, but we can't be sending out a ton of emails like this will.